tests/client_fixture.py

```diff
@@ -59,6 +59,20 @@ def profile_test_client():
     Base.metadata.drop_all(bind=test_engine)
 
 
+@pytest.fixture(scope="session")
+def weekly_tasks_test_client():
```

Comment:
Great! Maybe we should use it in more tests?

Reply:
the other functions (internal) are not client related functions
so it seems I can't use it on them.

Reply:
https://github.com/PythonFreeCourse/calendar/pull/146#discussion_r569155593

it seems the comment about the 'Depends(get_db)' encounters the same problem
the Depends is of no use if the function is not a router function.
I did not find a way to use it within a standard non-fastapi function

the internal functions are used by the routers functions which uses Depends

Reply:
I split the tests into two files: internal tests and route tests.

Tell me what you think about the rearrangement and my comments :)